getting-started/proc-add-source-control-rhdh-catalog.adoc

```diff
@@ -3,19 +3,59 @@
 
 To populate the Catalog in {product}, you need to add software templates, and to add the templates, you must enable a source control such as GitHub, GitLab, or BitBucket.
 
-== Setting GitHub integration and authentication
-
-The GitHub integration and authentication is required to enable the GitHub plugins in {product}.
-
 .Prerequisites
 
 * You have a GitHub account.
-* You have an account in Red Hat OpenShift
+* You have an account on the Red Hat OpenShift cluster.
+* You have installed the {product-short}, otherwise the GitHub login fails. For more information about installation, see xref:proc-install-rhdh-helm_rhdh-getting-started[].
+
+== Configuring GitHub authentication
+
+The configuration of GitHub authentication is required to enable the GitHub OAuth login in {product-short}.
 
 .Procedure
 
-. In OpenShift, click *ConfigMap*.
-. Modify your `app-config-rhdh` file to include the GitHub configuration as follows:
+. In the Red Hat OpenShift cluster, navigate to the main page of the GitHub organization where you want to create the OAuth application. 
+. Click *Settings* -> *<> Developer Settings* -> *OAuth Apps* -> *Register an application*
+. Enter the application name as `Developer Hub`.
+. Add the following as the *Homepage URL*:
++
+--
+`https://developer-hub-<NAMESPACE_NAME>.<OPENSHIFT_ROUTE_HOST>/`
+--
+
+. Add the following URL as *Authorization callback URL*:
++
+--
+`https://developer-hub-<NAMESPACE_NAME>.<OPENSHIFT_ROUTE_HOST>/api/auth/github/handler/frame`
+--
+
+. Uncheck the *Enable Device Flow* checkbox.
+. Click *Register application* to create your OAuth application.
+. After creating the application, click *Generate a new client secret* and copy the generated client secret.
+. In OpenShift, click *ConfigMaps*.
+. Generate a key/value secret named 'github-secrets' using the provided environment variables as keys, and input the values you generated for your GitHub OAuth application:
++
+--
+.. In Red Hat OpenShift, go to the *Secrets* tab and click *Create*.
+.. Select *Key/value secret*.
+.. Enter *Secret name* as `github-secrets`.
+.. Add environment variables as *Key* and *Value* and click *Create*.
+
+.Environment variables
+[cols="1,1"]
+|===
+|Key |Value 
+
+|GITHUB_APP_CLIENT_ID
+|Client ID from OAuth application
+
+|GITHUB_APP_CLIENT_SECRET
+|Client Secret from OAuth application
+|===
+--
```

Comment:
Unclear what this part represents. If this is a result, adding the .Result heading might better explain what this is showing to the user